### DISCUSSION TITLE
JEI & JER tweaks from MIA

### DISCUSSION THREAD

**Discussion by xJon** [ACGaming/UniversalTweaks#120] [Tweak Suggestions]
Beyond https://github.com/ACGaming/UniversalTweaks/discussions/119, JEI/JER are quite *universal* mods and adding tweaks specifically to them is unlikely to increase complexity whilst being guaranteed to provide value.

**Comment by Turkey9002** [comment]
It would be nice to have something similar to [this](https://twitter.com/VazkiiMods/status/1564706757976498176) as a well since JEI already has a section to display all fuels.

**Comment by xJon** [comment]
Closing this as from experience MIA is not too intrusive and would have a lot of features to reimplement, so it's not a priority.
